tests/devices/i19/test_beamstop.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+
+from dodal.devices.i19.beamstop import BeamStop
+
+
+@pytest.fixture
+async def beamstop() -> BeamStop:
+    beamstop = BeamStop("", "mock_beamstop")
+    await beamstop.connect(mock=True)
+
+    return beamstop
+
+
+def test_beamstop_can_be_created(beamstop: BeamStop):
```

Comment:
Could: I've started smoke testing devices like these using `assert_reading` e.g. see https://github.com/DiamondLightSource/dodal/pull/1144, which I guess also tests the hints too. But I don't think there's much in it